<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
	<class>TaskBookClass</class>
	<widget class="QWidget" name="TaskBookClass">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>743</width>
				<height>366</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>TaskBook</string>
		</property>
		<layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
			<item>
				<layout class="QVBoxLayout" name="left_column">
					<item>
						<widget class="QPushButton" name="qpbSettings">
							<property name="text">
								<string>Settings</string>
							</property>
						</widget>
					</item>
					<item>
						<spacer name="vertspacer3">
							<property name="orientation">
								<enum>Qt::Vertical</enum>
							</property>
							<property name="sizeHint" stdset="0">
								<size>
									<width>20</width>
									<height>40</height>
								</size>
							</property>
						</spacer>
					</item>
					<item>
						<widget class="QPushButton" name="qpbAddTask">
							<property name="text">
								<string>AddTask</string>
							</property>
						</widget>
					</item>
				</layout>
			</item>
			<item>
				<widget class="QTabWidget" name="categories">
					<property name="tabPosition">
						<enum>QTabWidget::North</enum>
					</property>
					<property name="tabShape">
						<enum>QTabWidget::Rounded</enum>
					</property>
					<property name="currentIndex">
						<number>0</number>
					</property>
					<property name="elideMode">
						<enum>Qt::ElideLeft</enum>
					</property>
					<property name="usesScrollButtons">
						<bool>true</bool>
					</property>
					<property name="documentMode">
						<bool>false</bool>
					</property>
					<property name="tabsClosable">
						<bool>true</bool>
					</property>
					<property name="movable">
						<bool>true</bool>
					</property>
					<property name="tabBarAutoHide">
						<bool>false</bool>
					</property>
					<widget class="QWidget" name="qwAll">
						<attribute name="title">
							<string>All</string>
						</attribute>
						<layout class="QVBoxLayout" name="verticalLayout">
							<item>
								<widget class="QScrollArea" name="qsaArea">
									<property name="verticalScrollBarPolicy">
										<enum>Qt::ScrollBarAlwaysOn</enum>
									</property>
									<property name="horizontalScrollBarPolicy">
										<enum>Qt::ScrollBarAlwaysOff</enum>
									</property>
									<property name="widgetResizable">
										<bool>true</bool>
									</property>
									<widget class="QWidget" name="qwArea">
										<property name="geometry">
											<rect>
												<x>0</x>
												<y>0</y>
												<width>513</width>
												<height>299</height>
											</rect>
										</property>
										<layout class="QVBoxLayout" name="verticalLayout_2">
											<item>
												<spacer name="vertSpacer">
													<property name="orientation">
														<enum>Qt::Vertical</enum>
													</property>
													<property name="sizeHint" stdset="0">
														<size>
															<width>20</width>
															<height>40</height>
														</size>
													</property>
												</spacer>
											</item>
										</layout>
									</widget>
								</widget>
							</item>
						</layout>
					</widget>
				</widget>
			</item>
			<item>
				<layout class="QVBoxLayout" name="rigthColumn" stretch="0,0">
					<property name="spacing">
						<number>6</number>
					</property>
					<property name="leftMargin">
						<number>0</number>
					</property>
					<property name="rightMargin">
						<number>0</number>
					</property>
					<property name="bottomMargin">
						<number>0</number>
					</property>
					<item>
						<widget class="QPushButton" name="qpbAddCategory">
							<property name="text">
								<string>AddCategory</string>
							</property>
						</widget>
					</item>
					<item>
						<spacer name="vertSpacer2">
							<property name="orientation">
								<enum>Qt::Vertical</enum>
							</property>
							<property name="sizeHint" stdset="0">
								<size>
									<width>20</width>
									<height>40</height>
								</size>
							</property>
						</spacer>
					</item>
				</layout>
			</item>
		</layout>
	</widget>
	<layoutdefault spacing="6" margin="11"/>
	<resources>
		<include location="taskbook.qrc"/>
	</resources>
	<connections/>
</ui>